src/modules/orders/services/notification.services.ts

```
@@ -0,0 +1,80 @@
+    await this.smsSender({ to: phone, message });
+  }
+
+  private async sendMail(EmailDto: EmailDto) {
+    try {
+      await sgMail.send({
+        to: EmailDto.to,
+        from: this.configService.get<string>('SENDGRID_VERIFIED_SENDER'),
+        subject: EmailDto.subject,
+        html: EmailDto.html,
+      });
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)